train.py

```diff
@@ -219,17 +219,16 @@ def loss_fn(pred, labels):
 
     metric_logger = build_metric_logger(job_config)
 
-    # torch.compile model for improved performance
     if job_config.training.compile:
```

Comment:
nit: since we are moving compiling to parallelize_llama, let's add some comment on L201.